recipe/meta.yaml

```
@@ -23,27 +23,34 @@ requirements:
     - setuptools
     - wheel
     - cython
-    - fftw
     - {{ pin_compatible('numpy') }}
+    - fftw 3.3.9
   run:
+    - python
     - _openmp_mutex  # [linux]
     - llvm-openmp    # [osx]
-    - fftw
+    - fftw >=3
```

```suggestion
    - fftw # bound set through run_exports
```
see: https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Flinux-64%2Ffftw-3.3.9-h5eee18b_2.conda